dgbpy/uikeras.py

```diff
@@ -102,20 +102,40 @@ def getUiPars(uipars=None):
   decimateCB( uiobjs['dodecimatefld'].active,uiobjs['chunkfld'],uiobjs['sizefld'] )
   return uipars
 
+def getAdvancedUiPars(uipars=None):
+  uiobjs={}
+  if not uipars:
+    uiobjs = {
+      'augmentfld': CheckboxGroup(labels=['Enable Data Augmentation'], visible=True, margin=(5, 5, 0, 5)),
+      'tensorboardfld': CheckboxGroup(labels=['Enable Tensorboard'], visible=True, margin=(5, 5, 0, 5)),
+      'cleartensorboardfld': CheckboxGroup(labels=['Clear Tensorboard log files'], visible=True, margin=(5, 5, 0, 5))
+    }
+    parsgrp = column(*list(uiobjs.values()))
+    uipars = {'grp':parsgrp, 'uiobjects':uiobjs}
+  else:
+    uiobjs=uipars['uiobjects']
+  uiobjs['augmentfld'].active = []
```

Comment:
You must read in the dgbkeras.dict object to retrieve the appropriate default values.
See that it is also the very first line of getUiPars